Pipfile

```diff
@@ -8,6 +8,8 @@ cryptography = "==3.4.8"
 crcmod = "==1.7"
 protobuf = "==3.20.3"
 psutil = "==5.7.0"
+pyright = "*"
+mypy-protobuf = "*"
```

Comment:
Think this needs to go into dev-packages instead.

Reply:
Good point, will fix.

Reply:
Had to upgrade `nodeenv` as well to make it work, for some reason. Hopefully that's ok.

Reply:
Should be.